server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java

```diff
@@ -311,7 +311,7 @@ public class ConnectionConfig
     @JsonIgnore
     private OBInstanceRoleType instanceRoleType;
 
-    @JsonIgnore
+    // @JsonIgnore
```

Comment:
we'd better not remove this

Reply:
you can add getter method if you want to return some other field by json

Reply:
😊modified to @Getter